app/assets/stylesheets/jquery-ui/custom-theme/jquery-ui-1.8.13.custom.scss

```diff
@@ -89,14 +89,14 @@
 ----------------------------------*/
 
 /* states and images */
-.ui-icon { width: 16px; height: 16px; background-image: url(images/ui-icons_222222_256x240.png); }
-.ui-widget-content .ui-icon {background-image: url(images/ui-icons_222222_256x240.png); }
-.ui-widget-header .ui-icon {background-image: url(images/ui-icons_ffffff_256x240.png); }
-.ui-state-default .ui-icon { background-image: url(images/ui-icons_888888_256x240.png); }
-.ui-state-hover .ui-icon, .ui-state-focus .ui-icon {background-image: url(images/ui-icons_454545_256x240.png); }
-.ui-state-active .ui-icon {background-image: url(images/ui-icons_454545_256x240.png); }
-.ui-state-highlight .ui-icon {background-image: url(images/ui-icons_2e83ff_256x240.png); }
-.ui-state-error .ui-icon, .ui-state-error-text .ui-icon {background-image: url(images/ui-icons_cd0a0a_256x240.png); }
+.ui-icon { width: 16px; height: 16px; background-image: image_path('ui-icons_222222_256x240.png'); }
+.ui-widget-content .ui-icon {background-image: image_path('ui-icons_222222_256x240.png'); }
+.ui-widget-header .ui-icon {background-image: image_path('ui-icons_ffffff_256x240.png'); }
+.ui-state-default .ui-icon { background-image: image_path('ui-icons_888888_256x240.png'); }
+.ui-state-hover .ui-icon, .ui-state-focus .ui-icon {background-image: image_path('ui-icons_454545_256x240.png'); }
```

Comment:
Name of function `image_path` should be written in all lowercase letters with hyphens instead of underscores<br>Each selector in a comma sequence should be on its own single line